wandb/sdk/wandb_artifacts.py

```diff
@@ -270,6 +270,13 @@ def name(self) -> str:
 
         return self._name
 
+    @property
+    def full_name(self) -> str:
+        if self._logged_artifact:
+            return self._logged_artifact.full_name
+
+        return f"{self.entity}/{self.project}/{self.name}"
```

Comment:
If you make the change suggested above this becomes
```suggestion
        return super().full_name
```